src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -33,7 +33,7 @@ function PageCard({ intl, page }) {
   );
 
   const handleClick = () => {
-    history.push(`${pathname}/${page.id}`);
+    history.push(`${pathname}${page.id}`);
```

Comment:
The path name already contains a slash. I was getting a double slash which was messing up routing.